internal/common/awsxray/testdata/segmentValidationFailed.txt

```diff
@@ -0,0 +1,30 @@
+{"format": "json", "version": 1}
```

Comment:
Like my below comment, this format is only used by the receiver, not exporter, so it's a bit confusing to include it here.